.docksal/docksal.yml

```diff
@@ -10,63 +10,11 @@ services:
       - MIGRATE_DB_ENV
     volumes:
       - ${PROJECT_ROOT}/.docksal/etc/drush:/etc/drush:ro
-    # Expose port and let Storybook attach to it.
-    expose:
-      - 6006
-    labels:
-      - io.docksal.virtual-host=storybook.${VIRTUAL_HOST},storybook.${VIRTUAL_HOST}.*
-      - io.docksal.virtual-port=6006
 
-  # Fake email
-  mail:
+  solr:
+    image: "devkteam/solr:8"
     extends:
       file: ${HOME}/.docksal/stacks/services.yml
-      service: mail
-      
-  # PHPMyAdmin
-  pma:
```

Comment:
You're removing PMA?

Reply:
we have sequelace

Reply:
Some people do.

Reply:
we can make a video to show how to add sequelace.   it's better to add extra containers as needed instead of assuming people need this since it's not part of the default pantheon stack